lib/helpers/getNextSrcDir.js

```diff
@@ -0,0 +1,9 @@
+const { join } = require("path");
+
+const getNextSrcDirs = () => {
+  return ["pages", "src/pages", "public", "styles"].map((dir) =>
```

Comment:
`src/pages` per https://nextjs.org/docs/advanced-features/src-directory.

I couldn't figure out if theses dirs is something I can get from the `config` object.

Reply:
i dont think so @ getting from config, i think this is fine re: `src/pages`

should we include `"components"` which is a common pattern in next projects (see https://github.com/netlify/explorers/tree/main/src/components) and the main building blocks of /pages?

also, should we just include "src" instead of "src/pages" and "src/components" etc?

Reply:
Yes `src` is much better